bin/run_qc.py

```diff
@@ -32,6 +34,12 @@ def _parse_args():
     geno_val_parser.add_argument('--sample_id', required=True)
     geno_val_parser.set_defaults(func=run_genotype_validation)
 
+    sp_contamination_parser = subparsers.add_parser('contamination_check')
+    sp_contamination_parser.add_argument('--fastq_files', required=True, nargs='+', help='the fastq file pairs')
+    sp_contamination_parser.add_argument('--run_id', required=True)
```

Comment:
`--run_id` should be replace by `--working_dir` 
or better: make --working_dir optional and build it from `os.path.join(cfg[jobs_dir], sample_id`